core/adminserver/admin_server.go

```diff
@@ -277,7 +277,7 @@ func (s *Server) initialize(ctx context.Context, logTree, mapTree *tpb.Tree) err
 
 	// If the tree is empty and the map is empty,
 	// add the empty map root to the log.
-	if logRoot.GetTreeSize() != 0 ||
+	if logRoot.TreeSize != 0 ||
```

Comment:
Doesn't using getters protect from a nil object?

Reply:
logRoot used to be a proto. Now it is a go struct that is verified.